package.json

```diff
@@ -25,25 +25,25 @@
   "author": "Alvaro Cabrera <pateketrueke@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "glob": "^7.1.6",
+    "glob": "^8.0.3",
     "global-or-local": "^0.1.7",
-    "open": "^7.3.0",
+    "open": "^8.4.0",
     "pixelmatch": "^5.3.0",
     "pngjs": "^6.0.0",
-    "wargs": "^0.9.6"
+    "wargs": "^0.10.0"
   },
   "devDependencies": {
     "bili": "^5.0.5",
     "eslint": "^7.11.0",
     "eslint-config-airbnb-base": "^14.2.0",
     "eslint-plugin-import": "^2.22.1",
     "is-svg": ">=4.2.2",
-    "sirv-cli": "^1.0.8",
-    "somedom": "^0.4.19",
-    "testcafe": "^1.20.1"
+    "sirv-cli": "^1.0.14",
+    "somedom": "^0.4.20",
+    "testcafe": "^2.0.1"
   },
   "peerDependencies": {
-    "testcafe": "^1.9.4"
+    "testcafe": "*"
   },
   "engines": {
     "node": ">=10.0.0"
```

Comment:
I am not sure if the whole thing still works on v10, or even v12, please make sure we upgrade the required engine too if we're not longer backward-compatible.

On github action we're using node v14 for sure, we should other versions as well?

Reply:
good point - I checked and testcafe didn't change the node.js engine support
the change in peer dependencies let the user use any version of testcafe.
I run test e2e tests on node18
I'll try now with node 16 and node 14
then I'll update the engine to node 14 to match testcafe requirement https://github.com/DevExpress/testcafe/blob/master/package.json